web/pom.xml

```diff
@@ -100,6 +100,18 @@
         </dependency>
 
         <dependency>
+            <groupId>com.fasterxml.jackson.jaxrs</groupId>
```

Comment:
Cool. We will later need to find a way to return Task<...> from the  rest methods.